core/model/edition.py

```diff
@@ -724,6 +724,17 @@ def calculate_author(self):
             sort_author = " ; ".join(sorted(sort_names))
         else:
             sort_author = self.UNKNOWN_AUTHOR
+
+        def truncate_string(mystr: str):
+            if len(mystr) > 1024:
```

Comment:
It would helpful to use a constant here instead of hard coding 1024 in multiple places. We could do that either on the `Edition` class or a higher-level constant associated with PostgreSQL. This value is really tied to a limit on PG B-tree index row max length, so should probably be named accordingly.